app/styles/main.css

```diff
@@ -8069,7 +8073,7 @@ h5 {
 
 .input-group-addon.primary {
   color: white;
-  background-color: #3276b1;
```

Comment:
why is the change?

Reply:
I don't know what happened. It looks like the main.scss recompiled after I run grunt build.